gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ProcedureRequestMapper.java

```diff
@@ -104,6 +109,23 @@ private Annotation getNote(String text) {
         return null;
     }
 
+    private ProcedureRequestStatus getStatus(String planStatementText) {
+        if (planStatementText == null || planStatementText.isEmpty()) {
```

Comment:
I think you can use StringUtils in here which should be more concise option for checking for nulls and emptiness

Reply:
doh! Thats absolutely right, I'll fix that in future pull request touching this mapper :)